packages/hackathon/src/models/SheetRow.ts

```diff
@@ -57,4 +57,8 @@ export class SheetRow<T extends ISheetRow> extends RowModel<T> {
       user.permissions.has('update')
     )
   }
+
+  namespace() {
+    return 'hackathon'
+  }
```

Comment:
nice. this is the thing I was talking about yesterday (?). We should rename refactor to HackathonData and HackathonRow in the future.